pandas/core/dtypes/common.py

```diff
@@ -1651,7 +1651,7 @@ def _is_dtype_type(arr_or_dtype, condition) -> bool:
 
     try:
         tipo = pandas_dtype(arr_or_dtype).type
-    except (TypeError, ValueError, UnicodeEncodeError):
```

Comment:
is this a leftover-from-py2 thing?

Reply:
I don't know. `UnicodeEncodeError` is a sub-class of  `ValueError`.